cpp/daal/src/algorithms/kernel_inst_x86.h

```diff
@@ -72,8 +72,7 @@
         default:                                                                                                                                 \
         {                                                                                                                                        \
             using cntrTemplateInst = ContainerTemplate<__VA_ARGS__, sse2>;                                                                       \
-            static volatile services::internal::GpuSupportRegistrar<cntrTemplateInst> registrar;                                                 \
-            _cntr = (new cntrTemplateInst(daalEnv));                                                                                             \
+            _cntr                  = (new cntrTemplateInst(daalEnv));                                                                            \
```

Comment:
It seems like __DAAL_INSTANTIATE_DISPATCH_SYCL_IMPL macro can be removed and replaced with __DAAL_INSTANTIATE_DISPATCH_IMPL.

Reply:
The same applies to all kernel_inst_*.h files.